src/tmpfiles/tmpfiles.c

```diff
@@ -3446,7 +3446,8 @@ static int run(int argc, char *argv[]) {
                                 DISSECT_IMAGE_REQUIRE_ROOT |
                                 DISSECT_IMAGE_VALIDATE_OS |
                                 DISSECT_IMAGE_RELAX_VAR_CHECK |
-                                DISSECT_IMAGE_FSCK,
+                                DISSECT_IMAGE_FSCK |
+                                DISSECT_IMAGE_GROWFS,
```

Comment:
It seems a bit "magic" for this to happen in so many places… But OTOH, if we're mounting the fs rw, and already running fsck, growing the file systems seems like a natural extension.

Reply:
yeah it *is* magic. I initially wanted to do this entirely implicitly, like the fsck thing. But because it is so magic, and is possibly an invasive operation I made it opt-in, via the gpt flag.